apps/api/src/hyp3_api/validation.py

```diff
@@ -26,6 +26,10 @@ class GranuleValidationError(Exception):
     pass
 
 
+class DateValidationError(Exception):
```

Comment:
If you want to add a new `DateValidationError` exception type, then it will need to be handled in [`handlers.post_jobs`](https://github.com/ASFHyP3/hyp3/blob/develop/apps/api/src/hyp3_api/handlers.py#L34), otherwise it will just result in an Internal Server Error. (This is something that integration tests would catch!)

On the other hand, I don't think that having different types of validation exceptions actually provides any benefit, since they all show up as `400 Bad Request` for the user, but I haven't gotten around to refactoring to combine the three existing types (`BoundsValidationError, GranuleValidationError, MultiBurstValidationError`) into a single exception type. So I'm fine with adding a new exception type for now.